tests/src/test/scala/cats/tests/StdTests.scala

```diff
@@ -20,4 +20,7 @@ class StdTests extends FunSuite with Discipline {
   checkAll("Option[Int]", MonadFilterTests[Option].monadFilter[Int, Int, Int])
   checkAll("Option[String]", MonadFilterTests[Option].monadFilter[String, Int, Int])
   checkAll("List[Int]", MonadFilterTests[List].monadFilter[Int, Int, Int])
+  checkAll("List[Int]", MonoidKTests[List].identity[Int])
+  checkAll("Stream[Int]", MonoidKTests[Stream].identity[Int])
+  checkAll("Vector[Int]", MonoidKTests[Vector].identity[Int])
```

Comment:
Is `.identity` the right thing here? I would think it would be something like `.monoidK`, which would test all `MonoidK` laws, including those inherited from `SemigroupK`.